test/jdk/tools/jpackage/helpers/jdk/jpackage/test/MacHelper.java

```diff
@@ -334,7 +334,7 @@ static Path getInstallationDirectory(JPackageCommand cmd) {
             installLocation = cmd.getArgumentValue("--install-dir", () -> defaultInstallLocation, Path::of);
         }
 
-        return installLocation.resolve(cmd.name() + (cmd.isRuntime() ? "" : ".app"));
+        return installLocation.resolve(cmd.name() + (cmd.isRuntime() ? ".jdk" : ".app"));
```

Comment:
Why the ".jdk" suffix and not ".app"?

Reply:
Because JDK bundles has ".jdk" suffix. Why it should be ".app"?